shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/java/org/apache/shardingsphere/sql/parser/MySQLVisitor.java

```diff
@@ -249,7 +250,17 @@ public ASTNode visitShowLike(final ShowLikeContext ctx) {
     public ASTNode visitCreateUser(final CreateUserContext ctx) {
         return new DCLStatement();
     }
-
+    
+    @Override
+    public ASTNode visitDropRole(DropRoleContext ctx) {
+        return new DCLStatement();
+    }
+    
+    @Override
+    public ASTNode visitSetDefaultRole(SetDefaultRoleContext ctx) {
```

Comment:
Is it necessary to add? I could find this one in `META-INF/parsing-rule-definition/mysql/sql-statement-rule-definition.xml`

Reply:
setDefaultRole have add to sql-statement-rule-definition.xml，please review again